src/aiida/tools/query/formatting.py

```diff
@@ -60,7 +80,7 @@ def format_process_state(process_state: str) -> str:
     :param process_state: The process state.
     :return: String representation of process state.
     """
-    return f'{process_state.capitalize() if process_state else None}'
+    return f'{process_state.capitalize()}'
```

Comment:
I did not like that this can return a None. I checked this function is no where used outside this file. If one wants the old behavior, then at least I would adapt the typehints

Reply:
Comment: it's not returning a `None`, but the string `"None"`, right? so no need to fix type hints?

Reply:
Yeah, but the argument `process_state` to the function can be `None` so it should be
```python
def format_state(process_state: str | None, ...) -> str:
```